src/print.jl

```diff
@@ -436,6 +436,14 @@ function _nl_subexpression_string(print_mode, model::Model)
     return strings
 end
 
+anonymous_name(::Any, x::AbstractVariableRef) = "noname"
+
+anonymous_name(::Type{REPLMode}, x::VariableRef) = "_[$(index(x).value)]"
+
+function anonymous_name(::Type{IJuliaMode}, x::VariableRef)
+    return "\\_\\[$(index(x).value)\\]"
```

Comment:
@blegat we escape the underscore, so it'll get printed as `_[1]`

Reply:
Probably it should be `{\\_}_{$(index(x).value)}` so that it prints as an index like other variables ?